src/notation/view/widgets/editstaff.cpp

```diff
@@ -213,7 +213,7 @@ void EditStaff::updateInstrument()
 
     longName->setPlainText(m_instrument.nameAsPlainText());
     shortName->setPlainText(m_instrument.abbreviatureAsPlainText());
-    instrumentName->setText(m_instrument.nameAsPlainText());
+    instrumentName->setText(m_instrument.trackName());
```

Comment:
This appears to _almost_ work. You'll see the problem if you add a transposing instrument (actually two of them).

![image](https://user-images.githubusercontent.com/11011881/228048946-e2b1c5fb-d020-49e1-ab1f-3ba8a9340fbd.png)

We should see "Instrument: Clarinet in C" but instead we only see "Instrument: Clarinet".

Now the thing is, `trackName()` should include the transposition, so the fact it doesn't is a bug. However, it should also include the instrument number (in this case "2"), which means `trackName()` isn't the right tool to be using here.

But the bigger problem is that `trackName()` is an MSCX string rather than an internal string.

Reply:
It took me a sec to figure out how to access the templates, since the repository functions aren't static, but I think I've done it! Doing the steps you specified for the German instrument names now produces the correct localized strings. If you could give this one a look-over one more time I'd be very grateful!